.github/workflows/bot.yml

```diff
@@ -20,14 +20,6 @@ jobs:
             spark: "spark2"
           - scala: "scala-2.11"
             spark: "spark2,spark-shade-unbundle-avro"
-          - scala: "scala-2.12"
```

Comment:
@xushiyan I find it very hard to make it compatible between spark 3.2.0 and spark 3.0.x/spark3.1.x after we upgrade spark version to spark 3.2.0, so I commented out the workflow.

Reply:
@leesf  We introduce build profiles `spark3.0.x` and `spark3.1.x` mostly due to spark's own incompatibilities. Here I think we can make some rules: to enable v2 writer, users have to make sure they're on spark 3.2+. Sounds good? In future, we may gradually drop support for old spark versions if the old spark code deviates too far from the latest one.

Reply:
sounds good.